build.sbt

```diff
@@ -94,13 +95,38 @@ lazy val core = crossProject(JSPlatform, JVMPlatform)
     libraryDependencies ++= Seq(
       "org.typelevel" %%% "cats-core" % catsVersion,
       "org.typelevel" %%% "cats-effect-kernel" % catsEffectVersion,
+      "org.typelevel" %%% "cats-effect-std" % catsEffectVersion,
       "org.scodec" %%% "scodec-bits" % scodecBitsVersion,
       "org.scalameta" %%% "munit" % munitVersion % Test,
       "org.typelevel" %%% "cats-laws" % catsVersion % Test,
       "org.typelevel" %%% "cats-effect" % catsEffectVersion % Test,
       "org.typelevel" %%% "discipline-munit" % disciplineMUnitVersion % Test,
       "org.typelevel" %%% "munit-cats-effect-3" % munitCEVersion % Test
-    )
+    ),
+    Test / sourceGenerators += Def.task {
+      val scalafmtConfigFile = scalafmtConfig.value
+      val sharedTestResourceDir =
+        (ThisBuild / baseDirectory).value / "core/shared/src/test/resources"
+      val inputs = FileTreeView
+        .default
+        .list(Glob(sharedTestResourceDir / "CBC*256.rsp"))
+        .map(_._1.toFile)
+        .toSet
+
+      val generator = FileFunction.cached(
+        streams.value.cacheDirectory / "cbcTestVectors",
+        inStyle = FileInfo.hash,
+        outStyle = FileInfo.exists) { inputs =>
+        AESCBCTestVectorGenerator.generate(
+          inputs,
+          (Test / sourceManaged).value,
+          scalafmtConfigFile
+        )
+      }
```

Comment:
I think this could be improved by tracking the sources of the generator itself

Reply:
Nice! Although this setup seems different than what I've done in the past. Could we use the `fileInputs` key here instead?

https://github.com/armanbilge/gcp4s/blob/6263842d44b187f4e587d5e351cb6e47f164c772/project/DiscoveryPlugin.scala#L31

Reply:
Hmm there really is no end of ways to do the same thing in sbt is there? I haven't seen this `fileInputs` thing before, will take a look.